files/find.py

```diff
@@ -313,7 +313,7 @@ def main():
                        continue
 
                     try:
-                        st = os.stat(fsname)
+                        st = os.lstat(fsname)
```

Comment:
It depends. As path is always the actual symlink and not the file that it points to, I would expect the modes etc. to belong to that symlink (path) aswell. 
The other behaviour could be desired in some cases too, though. (Follow symlinked directories and list modes of the symlinked target - maybe change "path" too?)
Maybe two config settings, whether to follow symlinked directories and a second one, whether to follow symlinked files make sense?